build.sc

```diff
@@ -49,15 +48,9 @@ trait CommonModule extends ScalaModule with SbtModule with PublishModule {
     MavenRepository("https://oss.sonatype.org/content/repositories/releases")
   )
 
-  private def scalacCrossOptions = majorVersion match {
-    case i if i < 12 => Seq()
-    case _           => Seq("-Xsource:2.11")
-  }
+  private def scalacCrossOptions =  Seq("-Xsource:2.13")
```

Comment:
We shouldn't be using `-Xsource` at all, it's intended only for code migration, not for production.

Reply:
Removed it